#!/usr/bin/env python
PACKAGE = "cnbiros_shared_navigation"

import math
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_angle",	double_t, 0, "Minimum sector angle", -math.pi/2.0 , -math.pi/2.0, 0.0)
gen.add("max_angle",	double_t, 0, "Maximum sector angle",  math.pi/2.0 , 0.0, math.pi/2.0)
gen.add("num_sectors", int_t, 0,  "Number of sectors",	9, 0, 180)

exit(gen.generate(PACKAGE, "point_sector_parameters", "PointSector"))
